lib/iris/tests/integration/experimental/test_cube_to_poly.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+
+from iris import load_cube
+from iris.experimental.geovista import cube_faces_to_polydata
+from iris.experimental.ugrid import PARSE_UGRID_ON_LOAD
+from iris.tests import get_data_path
+
+
+def test_integration_2d():
+    pass
+
+
+def test_integration_1d():
+    file_path = get_data_path(
+        [
+            "NetCDF",
+            "unstructured_grid",
+            "lfric_surface_mean.nc",
```

Comment:
Pretty sure this file also describes a mesh? You need a file with standard 1d coordinates (rectilinear coordinates).